kernels/volk/volk_8u_conv_k7_r2puppet_8u.h

```diff
@@ -155,9 +155,12 @@ static inline void volk_8u_conv_k7_r2puppet_8u_spiral(unsigned char* syms, unsig
     once = 0;
   }
 
-    //unbias the old_metrics
+  //unbias the old_metrics
   memset(X, 31, d_numstates);
 
+  // initialize decisions
+  memset(D, 0, (d_numstates/8) * (framebits + 6));
```

Comment:
I think it makes sense to 0 these. `volk_alloc` doesn't guarantee that the allocated memory is 0'd, and since this variable isn't set to anything otherwise, 0 seems wise!